.github/workflows/ci-uto.yaml

```diff
@@ -33,7 +33,7 @@ env:
   SNOWFLAKE_ACCOUNT_NAME: ${{secrets.SNOWFLAKE_ACCOUNT_NAME}}
   SNOWFLAKE_SCHEMA: ASTROFLOW_CI
   SNOWFLAKE_DATABASE: SANDBOX
-  SNOWFLAKE_WAREHOUSE: ROBOTS
+  SNOWFLAKE_WAREHOUSE: DEMO
   SNOWFLAKE_HOST: https://gp21411.us-east-1.snowflakecomputing.com
```

Comment:
this host needs to be in secrets - dont want to expose it

Reply:
Thanks for pointing this out. Updated this is code.